src/conditions/check_fields.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
 use indexmap::IndexMap;
 use regex::Regex;
 use serde::{Deserialize, Serialize};
+use std::vec::Vec;
```

Comment:
Is this actually needed? This is part of the standard prelude.

Reply:
Yeah you're right -- removing it!